docs/codeflow/working-in-codeflow-ide.md

```diff
@@ -108,4 +111,63 @@ Check in the terminal if the dev server is still running. If you want to restart
 
 ### Reopening the Preview panel 
 
-If you close the Preview by accident, you can reopen it by selecting the icon of a plug entitled "Ports in use" from the left-side navigation bar. Note that you can open the Preview in a separate tab or as a split screen.
\ No newline at end of file
+If you close the Preview by accident, you can reopen it by selecting the icon of a plug entitled "Ports in use" from the left-side navigation bar. Note that you can open the Preview in a separate tab or as a split screen.
+
+## Running monorepos in Codeflow IDE
+
+Codeflow IDE supports workspaces. Follow a walkthrough below to get your monorepo running in Codeflow.
+
+### npm
+
+To define a workspace, add the `workspaces` field to the `package.json`:
+
+```
+"workspaces": [
+  "packages/*"
+],
+```
+
+This refers to every sub-directory inside `packages` which contains a `package.json`.
+
+Check [npm documentation](https://docs.npmjs.com/cli/v7/using-npm/workspaces) for more information. 
+
+
+### pnpm
+
+To configure a workspace with pnpm, add the `pnpm-workspace.yaml` file to the root of the project:
+
+```yaml
+packages:
+  - 'packages/*'
+```
+
+Next, add a package from the workspace as a dependency to another package. In the example below we define `frontend` as dependent on `common`:
```

Comment:
I think if people don't know what the project looks like, as in they don't have a link, they will be confused what `frontend` and `common` means and how it looks. Do we maybe want to link to the projects I created in my collection?

Reply:
Alternatively we can have a code block before `### npm` that shows the project structure? For example:
```
workspace-project
├─ packages
│  ├─ backend
│  │  ├─ package.json
│  │  └─ index.js
│  ├─ frontend
│  │  ├─ package.json
│  │  └─ index.js
│  └─ common
│     ├─ package.json
│     └─ index.js
└─ package.json
```

Reply:
Oh wow I totally forgot to link them - the beauty of multitasking. Thank you! Will add them + also the structure for folks who prefer not to context-switch :)

Reply:
Added the links to each setup + the collection, as well as the project structure :)